src/Runner.Worker/FileCommandManager.cs

```diff
@@ -182,6 +182,14 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
                     return;
                 }
 
+                if (fileSize > AttachmentSizeLimit)
```

Comment:
We should probably also remove this block where it is added in the if/else block below

Reply:
Removed in [674f15b](https://github.com/actions/runner/pull/2321/commits/674f15bb436e7ed6c46d805e8b2d57ce3ca6a100)!

Reply:
Beautiful!!